### DISCUSSION TITLE
3차 데모데이 이슈 미리 정해봐용

### DISCUSSION THREAD

**Discussion by kyum-q** [woowacourse-teams/2024-code-zap#180] [이야기해봐요]
# 이슈를 정해봅시다

## 개발

# 기간을 정해봅시다 

##⚡ 표시는 목요일이 마감기간

**Comment by kyum-q** [comment]
## 디자인 
- 마이페이지 디자인⚡
- 로그인 디자인⚡
- 회원가입 디자인⚡
- 헤더 반응형 대응⚡
- 다크모드에서 라이트모드로 변경⚡
- 영어를 한글로 변경⚡

**Comment by kyum-q** [comment]
## API 명세

- API 명세 재작성⚡
- 새 API 작성⚡
- ERD 재설계⚡
- Swagger 변경⚡

**Comment by kyum-q** [comment]
# 개발

**Reply by kyum-q** [reply]
## 인증인가
- 이메일 회원가입⚡
- 이메일 로그인⚡
- 로그아웃⚡

**Reply by kyum-q** [reply]
## 코드 템플릿
> 모든 조회에는 페이징 처리됨

- 템플릿 등록 - 카테고리, 태그 등록, 인증/인가 정보 추가 ⚡️
- 템플릿 조회 - 카테고리, 태그 정보 추가 ⚡️
- 템플릿 수정 - 카테고리 수정, 태그 등록 및 삭제, 인증/인가 정보 추가 ⚡️
- 템플릿 삭제 - 인증/인가 정보 추가 ⚡️

---
- 나의 전체 템플릿 조회 ⚡️
- 카테고리별 나의 전체 템플릿 조회 ⚡️
- 페이징 ⚡️
---
- 검색
- 태그 필터링
- 코드 템플릿 정렬

**Reply by kyum-q** [reply]
## 카테고리
- 카테고리 생성 ⚡️

**Reply by Hain-tain** [reply]
**- 단일 템플릿 CRUD : 헤인, 초롱 (헤롱팀)**
**- 전체 템플릿 : 월하, 짱수(월수팀)**

**Comment by kyum-q** [comment]
## 3차 스프린트 요구사항

**Reply by kyum-q** [reply]
### BE 

- 로깅⚡
- 모니터링

**Reply by kyum-q** [reply]
### FE 
- 로깅
- 모니터링

**Comment by kyum-q** [comment]
## BE 역할 분배

- 로깅 및 모니터링 (2) - 켬미 ❤️ 몰리
- 인증/인가 (1) - 제우스 ❤️ 무중단배포
- 템플릿 (2) - 초롱 ❤️ 짱수
> 알아서 분배해라

**Comment by Jaymyong66** [comment]
## FE 역할 분배

<7월 마지막주 task>
- 인증/인가 (1)
- 템플릿 (2)

<8월 첫째주 추가 task>
- 로깅 및 모니터링 (1)
- 인증/인가 (1)
- 템플릿 (1)
